js/adapt-assessmentArticleModel.js

```diff
@@ -198,6 +198,18 @@ const AssessmentModel = {
   _setupBanks() {
     const assessmentConfig = this.getConfig();
     const bankSplits = assessmentConfig._banks._split.split(',');
+
+    this.findDescendantModels('block')
+      .filter(block => block.findDescendantModels('question').length > 0)).forEach(block => {
+        if (isNaN(block._quizBankId) || block._quizBankId < 1) {
```

Comment:
Could you do variables for these expressions?
```js
const isInvalidNumber = (isNaN(block._quizBankId) || block._quizBankId < 1);
const isOutOfBounds = (block._quizBankId > bankSplits.length);
if (isInvalidNumber) logging.warn(`Bank ID ${block._quizBankId} is not a valid number`);
if (isOutOfBounds) logging.warn(`Bank ID ${block._quizBankId} exceeds the number of available splits (${bankSplits.length})`);
```

It just makes it much easier to read.

Reply:
block._quizBankId returns as undefined. Think we need const blockAssessmentConfig = block.get('_assessment'); before isInvalidNumber, then replace 'block._quickBankId' with 'blockAssessmentConfig._quizBankID'